src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint_legacy.py

```diff
@@ -137,6 +136,13 @@ def __init__(
     ):
         super().__init__()
 
+        deprecation_message = (
```

Comment:
Let's fully deprecate this class. Now "normal" inpaint has all features that are needed.